plugins/confighelper/plugin.js

```diff
@@ -16,9 +16,6 @@ function dataIsEmpty( data )
 	if ( !data)
 		return true;
 
-	if ( data.length > 20 )
-		return false;
-
 	var value = data.replace( /[\n|\t]*/g, '' ).toLowerCase();
 	if ( !value || value == '<br>' || value == '<p>&nbsp;<br></p>' || value == '<p><br></p>' || value == '<p>&nbsp;</p>' || value == '&nbsp;' || value == ' ' || value == '&nbsp;<br>' || value == ' <br>' || value == '<p class="fs-14"><br></p>' || value == '<p class="fs-14">&nbsp;<br></p>')
```

Comment:
@nodesocket We’re trying to limit our changes. This is the ckeditor source, which, sadly, we’ve had to fork, but we still want to maintain the minimal surface area of changes.